packages/integration-tests/tests/retrospective-demo/reflect.test.ts

```diff
@@ -54,6 +110,123 @@ test.describe('restrospective-demo / reflect page', () => {
     ).toBeVisible()
   })
 
-  // test('displays simulated users writing reflections in the stop column')
-  // test('displays simulated users writing reflections in the continue column')
+  test('displays simulated users writing reflections in the stop column', async ({page}) => {
+    test.setTimeout(60_000)
```

Comment:
This test takes quite a long time to run because the simulation runs slowly enough for people to see what's going on.

I figured it would be best for this test to take longer vs. trying to speed up the interactions in the dev/test environment. That said, I could see an argument to speed this up in test to reduce the overall runtime of the test suite.